### ISSUE TITLE
Build a tool that automates issue-maintenance between GitHub and Jira

### ISSUE THREAD

**Issue by mwroffo** [mwroffo/testrepo#71]
See the corresponding [repo|https://github.com/mwroffo/pantabot] and readme.

We want Zowe to be an open-source project; but keeping the open-source community updated on internal progress produces hours of tedium for project managers like [~nrogers] and Scrum Masters like [~rchowdhary] who have to fill out GitHub issues manually. This tool (eventually a bot, perhaps) will automate the process of creating GitHub issues by fetching metadata for selected Jira issues, converting them into GitHub-friendly issues, and posting them to a corresponding repo under the Zowe organization on GitHub.

MVP:
 [x] port a selected jira issue to github with a simple command-line interface. Hopefully mapping the statuses in both Jira and github
 [x] add a basic UI for usability

Also implemented:
[x] leverage local machine's keychain to securely store a user's github/jira credentials for persistent logins.

Next:
[x] Bulk milestone updates.
[ ] Robust unit and e2e test suites. 

Really nice to have:
 [ ] accomplish this in reverse: Issues created on a zowe github repo by open-source contributors automatically prompt [~sgrady] for inclusion in Rocket's internal Jira. (likely will be a separate Issue)
 [ ] fetch JSON directly from Jira rather than parsing XML (which requires only ordinary permissions). mroffo is waiting on RAC ticket for Jira API access
